provisioner/roles/install_configure_snort/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+  vars:
+    ids_config_snort_version: '2.9.13'
```

Comment:
Version, provider, user and group are now defined in the defaults/main.yml as part of this commit, we can remove them here.